tests/integrations/django/test_basic.py

```diff
@@ -21,6 +22,19 @@
 
 from tests.integrations.django.myapp.wsgi import application
 
+# Hack to prevent from experimental feature introduced in version `4.3.0` in `pytest-django` that
+# requires explicit database allow from failing the test
+pytest_mark_django_db_decorator = pytest.mark.django_db
+try:
+    pytest_version = tuple(map(int, pytest_django.__version__.split(".")))
+    if pytest_version > (4, 2, 0):
+        pytest_mark_django_db_decorator = pytest.mark.django_db(databases="__all__")
+except ValueError:
+    if "dev" in pytest_django.__version__:
+        pytest_mark_django_db_decorator = pytest.mark.django_db(databases="__all__")
+except AttributeError:
+    pass
```

Comment:
I think this is simpler:

```suggestion
pytest_mark_django_db_decorator = pytest.mark.django_db(databases="__all__")
try:
    pytest_version = tuple(map(int, pytest_django.__version__.split(".")))
    if pytest_version <= (4, 2, 0):
        pytest_mark_django_db_decorator = pytest.mark.django_db
except (ValueError, AttributeError):
    pass
```

you'd have to test if this works though

Reply:
Nah that would not work because `pytest_django.__version__` was introduced in `4.0` I think

Reply:
https://pytest-django.readthedocs.io/en/latest/changelog.html#v4-0-0-2020-10-16